test/fixtures/output/flink/artifact/create-help.golden

```diff
@@ -9,10 +9,11 @@ Create Flink artifact "my-flink-artifact".
   $ confluent flink artifact create my-flink-artifact --artifact-file plugin.jar
 
 Flags:
-      --artifact-file string   REQUIRED: Flink artifact JAR file.
-      --description string     Description of Flink artifact.
-      --context string         CLI context name.
-  -o, --output string          Specify the output format as "human", "json", or "yaml". (default "human")
+      --artifact-file string      REQUIRED: Flink artifact JAR file or ZIP file.
```

Comment:
We might consider renaming this to `--file` since it's easier to type.
* `confluent flink artifact create --artifact-file` is redundant
* `confluent flink artifact create --file` is not